docs/sources/reference/components/prometheus/prometheus.exporter.windows.md

```diff
@@ -45,24 +46,24 @@ Refer to the [Collectors list](#collectors-list) for the default set.
----------------|--------------------|------------------------------------------|---------
-dfsr           | [dfsr][]           | Configures the dfsr collector.           | no
-exchange       | [exchange][]       | Configures the exchange collector.       | no
-iis            | [iis][]            | Configures the iis collector.            | no
-logical_disk   | [logical_disk][]   | Configures the logical_disk collector.   | no
-msmq           | [msmq][]           | Configures the msmq collector.           | no
-mssql          | [mssql][]          | Configures the mssql collector.          | no
-network        | [network][]        | Configures the network collector.        | no
-physical_disk  | [physical_disk][]  | Configures the physical_disk collector.  | no
-printer        | [printer][]        | Configures the printer collector.        | no
-process        | [process][]        | Configures the process collector.        | no
-scheduled_task | [scheduled_task][] | Configures the scheduled_task collector. | no
-service        | [service][]        | Configures the service collector.        | no
-smb            | [smb][]            | Configures the smb collector.            | no
-smb_client     | [smb_client][]     | Configures the smb_client collector.     | no
-smtp           | [smtp][]           | Configures the smtp collector.           | no
-text_file      | [text_file][]      | Configures the text_file collector.      | no
+Hierarchy      | Name               | Description                                       | Required
+---------------|--------------------|---------------------------------------------------|---------
+dfsr           | [dfsr][]           | Configures the dfsr collector.                    | no
+exchange       | [exchange][]       | Configures the exchange collector.                | no
+iis            | [iis][]            | Configures the iis collector.                     | no
+logical_disk   | [logical_disk][]   | Configures the logical_disk collector.            | no
+msmq           | [msmq][]           | Configures the msmq collector.                    | no
+mssql          | [mssql][]          | Configures the mssql collector.                   | no
+network        | [network][]        | Configures the network collector.                 | no
+physical_disk  | [physical_disk][]  | Configures the physical_disk collector.           | no
+printer        | [printer][]        | Configures the printer collector.                 | no
+process        | [process][]        | Configures the process collector.                 | no
+scheduled_task | [scheduled_task][] | Configures the scheduled_task collector.          | no
+service        | [service][]        | Configures the service collector.                 | no
+smb            | [smb][]            | Configures the smb collector.                     | no
+smb_client     | [smb_client][]     | (Deprecated) Configures the smb_client collector. | no
```

Comment:
```suggestion
smb_client     | [smb_client][]     | Configures the smb_client collector. | no
```

Reply:
I don't see an smb_client collector anymore:
https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.smb_client.md
Is it not deprecated?

Reply:
The official name is [`smbclient`](https://github.com/prometheus-community/windows_exporter/blob/25d90212ce8a438e533b89ee98a7c6a3577ebb86/internal/collector/smbclient/smbclient.go#L18). 

https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.smbclient.md

Reply:
Ok thank you, I'll revert this change.